hubspot.com.hubspot-template-2.json

```diff
@@ -6,18 +6,17 @@
   "version": 1,
   "logoUrl":"https://domainconnect.org/wp-content/uploads/2017/02/HubSpot-768x223.png",
   "description":"Enables a domain to work with HubSpot",
-  "variableDescription":"v1 is the customer id.",
   "records": [
```

Comment:
Your travis build is failing; I think because you need to specify "hostRequired" : true in the template.  If you set a CNAME = @, the host must be passed in with the template.  This is because you can't CNAME a root domain.

I think this is why the build is failing

Reply:
Not only that. It is also not allowed (by DNS RFC) to have 2 CNAME RRs on the same host.

Reply:
Right. I missed this.  From our conversation, couple of things.  First, not sure why you needed two records in the template.  If I recall, you want multiple teams calling and applying the template.  Not sure why they can't use the same variable name in the template.  This being said, based on Pawel's catch, you are also trying to have multiple authorized users of your domain.  You probably want to switch to TXT records for this.  It is similar to what Google is doing with TXT records.